doc/new-book/configuration-security.adoc

```diff
@@ -122,14 +122,20 @@ For information about additional `sslProfile` attributes, see xref:router_config
 
 If you plan to use SASL to authenticate connections, you must first add the SASL attributes to the `router` entity in the router's configuration file. These attributes define a set of SASL parameters that can be used by the router's incoming and outgoing connections.
 
-// Do we need to say something here about only supporting Cyrus SASL?
-
 .Prerequisites
 
 Before you can set up SASL, you must have the following:
 
 * xref:generating_sasl_database[The SASL database is generated.]
 * xref:configuring_sasl_database[The SASL configuration file is configured.]
+* The Cyrus SASL plugin is installed for each SASL mechanism you plan to use.
```

Comment:
Do we want to talk here about the EXTERNAL SASL mechanism? EXTERNAL means that the communication channel is external to SASL

Reply:
In later sections, we talk about setting the SASL mechanism by using the saslMechanisms attribute (for example: https://github.com/apache/qpid-dispatch/blob/master/doc/new-book/configuration-security.adoc#adding_sasl_authentication_to_incoming_connection). Rather than list them all in the doc, we just point to the official list from the Cyrus SASL doc.

Do you think that's sufficient, or is there something particular about EXTERNAL that we should mention it as a prereq to setting up SASL?

Reply:
I think that is sufficient. This PR can be merged. Do you want me to merge it? Thanks.

Reply:
Yes, please do.